### PR TITLE
Making missing TimeSeries return NotFound instead of BadRequest.

### PR DESCRIPTION
**Pull request by rripken**
This should address #80 
I'm introducing a new runtime exception that controllers can catch to better deal with Not Found.  I think it will be an ongoing concern and I'm hoping an exception will simplify implementations.

Also taking out the exception that was being thrown by #75.

### CODE REVIEW

File: cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java [hunk 1]
```diff
@@ -221,31 +221,28 @@ public void getAll(Context ctx) {
...
+                    ctx.queryParamAsClass("pagesize", Integer.class).getOrDefault(defaultPageSize));
 
             String acceptHeader = ctx.header(Header.ACCEPT);
             ContentType contentType = Formats.parseHeaderAndQueryParm(acceptHeader, format);
 
             String results;
             String version = contentType.getParameters().get("version");
-            if(version != null && version.equals("2")) {
+            if(version != null && version.equals("2"))
+            {
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java:245]: Does our checkstyle file indicate { on the next line or is your IDE set for a different project?
(I'm not a die hard for either form it just seems odd that your more recent PRs have starting doing this.)

File: cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesDaoImpl.java [hunk 2]
```diff
@@ -232,19 +211,52 @@ protected TimeSeries getTimeseries(String page, int pageSize, String names, Stri
...
+		if(cause instanceof SQLException){
+			SQLException sqlException = (SQLException) cause;
+
+			// There may be better ways to figure out if the exception was b/c Not found.
+
+			// When we type in a garbage tsId the cause is a SQLException.  Its cause is an OracleDatabaseException.
+			String message = sqlException.getMessage();
+
+			// Note: The message I saw looked like this:
+			//ORA-20001: TS_ID_NOT_FOUND: The timeseries identifier "%1" was not found for office "%2"
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesDaoImpl.java:245]: The 20001 is the error code you want to check for. That's specifically "no timeseries with that name found"

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
This otherwise looks good.

File: cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java (hunk 1 above)
Comment (rripken) [reply cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java:245]: I updated an ide plugin.  Must have lost the settings.  I'll look into it.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**